nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/AbstractCommentManager.java

```diff
@@ -148,6 +142,12 @@ protected void setCommentPermissions(CoreSession session, DocumentModel document
         session.setACP(documentModel.getRef(), acp, true);
     }
 
+    protected void setFolderPermissions(CoreSession session, DocumentModel dm) {
```

Comment:
For historical reason, could you keep this method at the place it was before?

Reply:
I moved it back up to its place in the file so git doesn't see it as a new method